doc/blog-post/2017-07-xx-1.3.0-announcement.md

```diff
@@ -0,0 +1,160 @@
+
+```js
+let formatter;
+
+Globalize.locale( "en" );
+formatter = Globalize.dateToPartsFormatter({datetime: "short"});
+
+formatter( new Date( 2010, 10, 30, 17, 55 ) ).map(({type, value}) => {
+  switch ( type ) {
+    case "year": return `<strong>${value}</strong>`;
+    default: return value;
+  }
+}).join( "" );
+// > "11/30/<strong>10</strong>, 5:55 PM"
+```
+
+See [React Date Input](https://github.com/rxaviers/react-date-input) as a demo of a UI component for React optimized for i18n and a11y.
+
+| Localized and smart date input           | Feb 28 in `en`, `es`, `pt`, `de`, `zh`, `ko`, and `ar` |
+| ---------------------------------------- | ---------------------------------------- |
+| ![en](https://media.giphy.com/media/xUA7aZAUNINGP2jI4M/giphy.gif) | ![en-es-pt-de-zh-ko-ar](https://media.giphy.com/media/3og0ILQu0KxLRewJnW/giphy.gif) |
+
+## Dynamically Augment Date Skeletons
+
+> This change was contributed by Marat Dyatko @vectart and Artur Eshenbrener @Strate in #462 and #604.
+
+The style used to display a date format often vary depending on the application. CLDR offers data for certain presets like short, medium, long, and full vs. date, time, and datetime. It also offers data for a lot of different [date fields](http://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table) and their combinations, which are used by globalize to sintetize an open-ended list of custom formats (called skeletons). But, what's interesting is that it would be prohibitively large if CLDR provided data for every single possible combination. So, there's an algorithm specified by UTS#35 to deduce missing data from the requested format.
+
+For example, there are four different presets to format a date and they would generate `"7/1/17"` (short), `"Jul 1, 2017"` (medium), `"July 1, 2017"` (long), and `"Saturday, July 1, 2017"` (full). Although, let's pretend we want something different such as `"Jul 1"`. For that format we could use `{skeleton: "MMMd"}`. Internally, globalize finds a direct match (in CLDR) for that skeleton. Let's now pretend we want `"July 1"`. For that format we could use `{skeleton: "MMMMd"}`. Internally, globalize doesn't find a direct match (in CLDR) for this skeleton and it needs to use the data for `MMMd`, which in the case of English maps to `"MMM d"`, and then replace `MMM` with `MMMM` dynamically generating `"MMMM d"`. If we wanted `"07/01"` instead, we could use `{skeleton: "MMdd"}`. Internally, globalize doesn't find a direct match in CLDR for this skeleton and it needs to use the data for `Md`, which in the case of English maps to `"M/d"`, and then replace `M` wtih `MM` and `d` with `dd` dynamically generating `"MM/dd"`.
```

Comment:
I asked for examples above, which this does indeed deliver, but it seems to mix old and new features together. Its not clear to me which part is new, and which already existed before. That's fine for docs, but for a release blog post, it should clarify the difference.